src/scale.rs

```diff
@@ -253,11 +262,6 @@ impl<T: NumCast, Src, Dst> Scale<T, Src, Dst> {
     }
 }
 
-impl<Src, Dst> Scale<f32, Src, Dst> {
-    /// Identity scaling, could be used to safely transit from one space to another.
-    pub const ONE: Self = Scale(1.0, PhantomData);
```

Comment:
That's unfortunate. We could use some const ONE in WR. Can we keep it?

Reply:
I'd prefer to either not have a specific constant for f32 here or have a way for ONE to work with all types (that implement One). In WebRender we can use `Scale::identity()` or `Scale::new(1.0)`, the later working even in const contexts.

Reply:
ok, sure. If our `new` is const fn, this is a non-issue